### PR TITLE
Update server.yml

### PR DESCRIPTION
**Pull request by darthwade**
Make sure pidfile & logfile directory exists and have correct owner.

### CODE REVIEW

File: tasks/server.yml [hunk 1]
```diff
@@ -24,6 +24,17 @@
 - name: check if log file exists
   stat: path={{ redis_logfile }}
   register: logfile_stat
+  
+- name: Redis | Make sure pidfile directory exists and have correct owner
+  sudo: yes
+  sudo_user: 'root'
```
Comment (DavidWittman) [inline tasks/server.yml]: Remove these sudo options from the tasks; they should be set at the playbook level.

File: tasks/server.yml [hunk 2]
```diff
@@ -24,6 +24,17 @@
...
+  file: path={{redis_pidfile|dirname}} owner={{redis_user}} state=directory recurse=yes
```
Comment (DavidWittman) [inline tasks/server.yml]: Pad the inside of your braces with spaces, e.g. `{{ redis_pidfile|dirname }}`.

**Comment by DavidWittman** [comment]
Hey @darthwade, thanks for the PR. Once you address my in-line comments I can merge this.

**Comment by darthwade** [comment]
Ready.

**Comment by DavidWittman** [comment]
Thanks @darthwade. I'm going to prep some similar changes for Sentinel and get this merged.

**Pull request merged**